third_party/s2n-bignum/include/_internal_s2n_bignum.h

```diff
@@ -14,4 +14,8 @@
 #   endif
 #else
 #   define S2N_BN_SYM_PRIVACY_DIRECTIVE(name)  /* NO-OP: S2N_BN_SYM_PRIVACY_DIRECTIVE */
-#endif
\ No newline at end of file
+#endif
+
+#ifdef AWSLC_PRIVATE_BUILD
+    #include <openssl/boringssl_prefix_symbols.h>
+#endif
```

Comment:
Can you always include this header file? It should either be filled with the prefixes, or be an empty file if it's not a prefix build. 

Or is this file supposed to come from s2n-bignum and be generic? If so can you solve it by including it in S2N_BIGNUM_PREFIX_INCLUDE?

>-include file
>    Process file as if "#include "file"" appeared as the first line of the primary source file. However, the first directory searched for file is the preprocessor's working directory instead of the directory containing the main source file. If not found there, it is searched for in the remainder of the "#include "..."" search chain as normal.

Reply:
You can't because s2n-bignum is not always build in the context of AWS-LC